advanced/Entity Matching with SDK-experimental demo.ipynb

```diff
@@ -268,7 +274,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Add `complete_missing`"
+    "### Add additional key `ignore_missing_fields`"
```

Comment:
```suggestion
    "### Add `ignore_missing_fields`"
```
Then maybe add a sentence below "To not get not get an error if some items in `sources` or `targets` having missing values add `ignore_missing_fields=True`.